scripts/add-features.R

```diff
@@ -206,21 +209,28 @@ apply_diacritic_features <- function(feat_mat, feat_vec, base_row_nums,
         ## which features are valued for this diacritic?
         valued <- find_valued_feats(vec, ignore_cols)
         ## pre-modifiers
-        if (row < base_row_nums[1]) {
+        if (row_int < base_row_nums[1]) {
```

Comment:
this was working before because R allows things like `3 > "2"` but it makes me uneasy, hence the explicit cast to an int.